odoo/addons/base/models/ir_qweb.py

```diff
@@ -320,7 +320,9 @@ def get_file_info(file):
                 'media': None,
             }
 
-        addons = module_boot()
+        addons = []
```

Comment:
we have a problem before no need to have addons. We just used to  access read a particular ir.ui.view using its xml id. For now we can only make test pass if we create an ir.asset record. Bundles created via the manifest don't work when we are not in a request

Reply:
in previous implementation of _get_asset_content
module_boot was only useful to set get_modules_order in template rendering context and it was only set when request was set